testinfra/modules/package.py

```diff
@@ -165,7 +165,13 @@ def version(self):
 class RpmPackage(Package):
     @property
     def is_installed(self):
-        return self.run_test("rpm -q %s", self.name).rc == 0
+        result = self.run("rpm -q --quiet %s 2>&1", self.name)
+        if result.succeeded:
+            return True
+        elif result.failed and result.stdout == '':
```

Comment:
FYI: If return code is `1` but there is no output/error, it means the package is not installed (for that we used `--quiet` option).